WNPRC_Purchasing/src/client/components/LineItemsPanelInputs.tsx

```diff
@@ -67,8 +70,10 @@ export const UnitInput: FC<InputProps> = memo(props => {
 
 interface NumericInputProps {
     value: number;
-    onChange: (colName, value) => void;
+    onChange?: (colName, value) => void;
```

Comment:
If you're making this optional you probably want to change the calls below to `onChange?.(...)`

Reply:
good point, done